README.md

```
@@ -1,28 +1,59 @@
-# 🔨 Rock Breaker / Разбей Скалу
+# ⚡️ Strike V
 
-<img src="docs/hero.png" width="800" alt="Hero Banner">
+<img src="https://raw.githubusercontent.com/WarLikeLaux/strike-v/main/docs/hero.png" width="800" alt="Hero Banner">
 
-[![Status](https://img.shields.io/badge/status-active_development-orange?style=flat-square)]()
+[![Status](https://img.shields.io/badge/status-stable-green?style=flat-square)]()
```

markdownlint-cli2 (0.18.1)

No empty links

(MD042, no-empty-links)